transports/azure-storage-queues/configuration_batchsize_asqn_[9,].partial.md

```diff
@@ -0,0 +1,13 @@
+Starting from version 8.1.3 of the transport (and from version 7.5.6 for the 7.x version of the transport), the batch size is dynamically calculated based on the endpoints [message processing concurrency limit](/nservicebus/operations/tuning.md) unless explicitly specified. The batch size is calculated based on the following formula
```

Comment:
This section is identical between versions 8 and 9. What are the benefits of splitting it into the dedicated partial for v9?

Reply:
This is needed because of the new Nuget alias. 👍